src/components/Stories.js

```diff
@@ -145,6 +149,60 @@ const Stories = ({ authorId, followerId }) => {
     fetchCategories()
   }, [])
 
+  useEffect(() => {
+    const getPermissions = async () => {
+      if (!userId) return
+
+      const permissionResponse = await userStory.getPermissionsById(userId)
+      const permissions =
+        permissionResponse.data.data.user.access_role[0].permissions.map(
+          (item) => item.name
+        )
+
+      const updatedAllowed = permissions.includes('Update Story Status')
```

Comment:
Do we have this role included in the new documentation? I think I only saw for Edit story.

Reply:
Yes, I have added it

The third point under backend config contains it.
https://github.com/EOS-uiux-Solutions/user-story/blob/drag-drop-274/README.md#backend-configurations

Reply:
oh I missed it. thank you!